.github/workflows/ci.yml

```diff
@@ -15,6 +15,15 @@ jobs:
           - { ruby: '3.0', mongodb: '4.4', gemfile: 'mongoid-8.0' }
           - { ruby: '3.1', mongodb: '4.4', gemfile: 'mongoid-8.0' }
           - { ruby: '3.2', mongodb: '4.4', gemfile: 'mongoid-8.0' }
+          - { ruby: '3.1', mongodb: '4', gemfile: 'mongoid-9.0' }
+          - { ruby: '3.1', mongodb: '5', gemfile: 'mongoid-9.0' }
+          - { ruby: '3.1', mongodb: '6', gemfile: 'mongoid-9.0' }
+          - { ruby: '3.2', mongodb: '4', gemfile: 'mongoid-9.0' }
+          - { ruby: '3.2', mongodb: '5', gemfile: 'mongoid-9.0' }
+          - { ruby: '3.2', mongodb: '6', gemfile: 'mongoid-9.0' }
+          - { ruby: '3.3', mongodb: '4', gemfile: 'mongoid-9.0' }
+          - { ruby: '3.3', mongodb: '5', gemfile: 'mongoid-9.0' }
+          - { ruby: '3.3', mongodb: '6', gemfile: 'mongoid-9.0' }
```

Comment:
This is fine, but we don't really need to test every combination of Ruby and MongoDB and mongoid. I would only keep latest 2.x and 3.x, but it's NBD.